Add Elgato Key Light plugin

cozybuild opened the pull request:
### Checks
- [x] I tested my changes or release
- [x] I agree to the [terms of service](https://streamcontroller.github.io/docs/latest/plugin_dev/submitting/terms/)

cozybuild commented:
As it's my first plugin I have to fix a few things and add some nice icons - so this is currently WIP. I only have an Elgato Ring Light for testing but it _should_ work with other Elgato lights, too. I can extend it in the future.

Core447 commented:
Great work! Dial support is amazing! Just ping me when you're ready for the release. However, if you're ok with it I would suggest to just put it out there and get feedback as soon as possible. The only thing that missing for a release is the store thumbnail.

cozybuild commented:
@Core447 All right, I added a store Thumbnail (and fixed a few bugs). I couldn't find any min-size requirements in the wiki, I hope this one is fine. Will continue with the work in the next days (more icons + multi lamp support)

Core447 commented:
Great. The thumbnail works! Because python makes it a bit tricky to import from folders with points in their names, StreamController plugin ids use underscores instead. I created a [PR](https://github.com/cozybuild/StreamControllerElgatoKeyLight/pull/1) to your plugin to fix that. Just update the commit sha of this PR once you updated that.

cozybuild commented:
Thanks! PR is merged, I added a small fix/improvement as well and released 0.1.3
